bench/Sender/Main.hs

```diff
@@ -56,7 +57,7 @@ main = do
     loadLogConfig logsPrefix logConfig
     setLocaleEncoding utf8
 
-    Right transport_ <- TCP.createTransport "0.0.0.0" "3432" TCP.defaultTCPParameters
+    Right transport_ <- TCP.createTransport "0.0.0.0" "127.0.0.1" "3432" TCP.defaultTCPParameters
```

Comment:
Sounds good. I thought about this aready, but not familiar enough with nt-tcp internal code to do this fix